bedrock/mozorg/templates/mozorg/about/policy/transparency/index.html

```diff
@@ -0,0 +1,252 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}{{ _('Transparency') }}{% endblock %}
+{% block body_class %}sand about-transparency{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'about-transparency' %}
+{% endblock %}
+
+{% block js %}
+  <!--[if IE 9]>
+  {% javascript 'matchmedia_addlistener' %}
+  <![endif]-->
+  {% javascript 'about-transparency' %}
+{% endblock %}
+
+{% block content %}
+  <div class="data-header">
+    <img src="{{ static('img/mozorg/about/transparency/hero-transparency.svg') }}" alt="">
+  </div>
+  <main role="main">
+    <header class="title-bar">
+      <div class="content">
+        <h1>{{ _('Transparency') }}</h1>
+      </div>
+    </header>
+
+    <section class="section" id="introduction">
+      <div class="content">
+        <p>
+          {{ _('Transparency is a key part of how Mozilla approaches user trust.') }}
+          {{ _('As an open source project that relies on open development, we build transparency into the way we write our code.') }}
+          {{ _('Additionally, our product documentation and notices describe how our products work and how we handle user data.') }}
+        </p>
+
+        <p>
+          {{ _('With this transparency in mind, we intend to publish bi­-annual transparency reports that help provide additional transparency to government disclosures and takedown requests.') }}
+        </p>
+        </p>
+      </div>
+    </section>
+
+    <nav class="section nav-block">
+      <nav class="content nav-report three-up">
+        <ul>
+          <li><a href="#faq">{{ _('FAQ') }}</a></li>
+          <li><a href="#definitions">{{ _('Definitions') }}</a></li>
+          <li><a href="{{ url('mozorg.about.policy.transparency.jan-dec-2015') }}">{{ _('2015 Report') }}</a></li>
+        </ul>
+      </nav>
+
+{# Hide previous reports nav until we have a previous report to link to. Maybe move this to an include...
+      <nav class="content nav-previous">
+        <h3>{{ _('Previous Reports') }}</h3>
+        <ul>
+          <li><a href="#">January-December 2015</a></li>
+          <li><a href="#">January-June 2016</a></li>
+          <li><a href="#">August-December 2016</a></li>
+        </ul>
+      </nav>
+#}
```

Comment:
Yeah, it felt dirty and yet also semantically valid... the whole block is navigation, it just has distinct bits of navigation inside it. I guess it can be a `div` (or maybe an `aside`) with two `nav`s inside.